databricks/koalas/tests/test_dataframe.py

```diff
@@ -360,10 +360,43 @@ def test_missing(self):
                                         "Series.*{}.*not implemented".format(name)):
                 getattr(d.a, name)()
 
+    def test_isin(self):
+        df = pd.DataFrame({'a': [4, 2, 3, 4, 8, 6],
+                           'b': [1, 2, 9, 4, 2, 4],
+                           'c': ["one", "three", "six", "seven", "one", "5"]},
+                          index=[10, 20, 30, 40, 50, 60])
+
+        ddf = koalas.from_pandas(df)
+        self.assert_eq(ddf.isin([4, 'six']), df.isin([4, 'six']))
+        self.assert_eq(ddf.isin({"a": [2, 8], "c": ['three', "one"]}),
+                       df.isin({"a": [2, 8], "c": ['three', "one"]}))
+
+        msg = "'DataFrame' object has no attribute {'e'}"
+        with self.assertRaisesRegex(AttributeError, msg):
+            ddf.isin({"e": [5, 7], "a": [1, 6]})
+
+        msg = "Dataframe and Series are not supported"
+        with self.assertRaisesRegex(NotImplementedError, msg):
+            ddf.isin(df)
+
+    def test_isin_series(self):
+
+        s = pd.Series(['lama', 'cow', 'lama', 'beetle', 'lama', 'hippo'], name='animal')
+
+        ds = koalas.from_pandas(pd.DataFrame(s)).animal
```

Comment:
you should be able to do `ks.Series(s)` now.

Reply:
Yes, or `koalas.from_pandas(s)` should work as well.